### ISSUE TITLE
Strange, two similar files with mdita link, one works the other doesn't External

### ISSUE THREAD

**Issue by weiwu-zhang** [dita-ot/dita-ot#3909]
Note that there are 2 files with the same link style, one file is completely okay and another causes dita to err

File that works:

https://github.com/TokenScript/documents/blob/164ff61896f3216fcd5f7d574c73e828a549dfa3/src/concept/MagicLink.md?plain=1#L13

File that doesn't (note the link at the end of the line)
https://github.com/TokenScript/documents/blob/164ff61896f3216fcd5f7d574c73e828a549dfa3/src/concept/Attestation.md?plain=1#L3

## Expected Behavior
````
$ dita --project config/website.yaml -o out/website
$
````

## Actual Behavior
````
dita --project config/website.yaml -o out/website
[topicpull] file:/home/a/OxygenXMLEditor/TokenScript-Documents/src/concept/Attestation.md:xref:1: [DOTX031E][ERROR]: The file MagicLink.md is not available to resolve link information. 
[topicpull] file:/home/a/OxygenXMLEditor/TokenScript-Documents/src/concept/Attestation.md:xref:1: [DOTX031E][ERROR]: The file MagicLink.md is not available to resolve link information. 
````

## Possible Solution
Change this line
https://github.com/TokenScript/documents/blob/164ff61896f3216fcd5f7d574c73e828a549dfa3/src/concept/Attestation.md?plain=1#L3
By deleting the link in it, then it works.

Which shows the problem is related to the link in it (you can see the link at the end of that line).

## Steps to Reproduce

1. `git clone -b dita2md git@github.com:TokenScript/documents.git`
2. `cd documents`
3. `dita --project config/website.yaml -o out/website`

## Copy of the error message, log file or stack trace
Error message in the beginning of the issue.

## Environment

* DITA-OT version: 3.7.1
* Operating system and version:
  _Ubuntu Linux 20.04_
* How did you run DITA-OT?
  _`dita` command,_
* Transformation type:
  _HTML5_

**Comment by jelovirt** [comment]
Could you add the verbose (`-v`) log output? Seems you  reference that file with

```xml
<topicref href="concept/Attestation.md" format="mdita"/>
```

Which means you're processing the file as MDITA. Which means the first paragraph is treated as a `<shortdesc>`. Could you test what happens if you add a dummy paragraph before https://github.com/TokenScript/documents/blob/164ff61896f3216fcd5f7d574c73e828a549dfa3/src/concept/Attestation.md?plain=1#L3. Will the error go away? If so, then the bug is in topicpull build step.

**Comment by raducoravu** [comment]
@jelovirt  I prepared a smaller dita project sample on which the problem can be reproduced
[mdShortdesc.zip](https://github.com/dita-ot/dita-ot/files/8377461/mdShortdesc.zip)
The DITA topic which links to the markdown files is in a folder outside of the folder where the markdown files are located, and when the shortdesc gets expanded the xref inside the shortdesc has a path which is not updated relative to the DITA topic.
Might be a duplicate of #3078

I think in the user's case if the "Bootstrap.dita" was moved in the same folder with the markdown topics Attestation and MagicLink, the problem would no longer occur.
But the problem is not severe, probably a tooltip when hovering those links does not contain the entire information.
